### PR TITLE
Eslint plugin style fixes

### PR DESCRIPTION
**Pull request by jsnoble**

### CODE REVIEW

File: jest.config.base.js [hunk 1]
```diff
@@ -62,31 +62,33 @@ export default (projectDir) => {
         watchPlugins: ['jest-watch-typeahead/filename', 'jest-watch-typeahead/testname'],
         workerIdleMemoryLimit: '200MB',
         testTimeout: 60 * 1000,
-        globals:  {
-            availableExtensions: ['.js', '.ts', '.mjs', 'cjs']
+        globals: {
+            availableExtensions: ['.js', '.ts', '.mjs', 'cjs'],
+            window: {}
```
Comment (busma13) [inline jest.config.base.js:67]: I think this is our issue with the worker-spec test.

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
         "k8s:rebuild": "ts-scripts k8s-env --rebuild",
         "k8s:restart": "ts-scripts k8s-env --rebuild --skip-build",
         "k8sV2": "TEST_ELASTICSEARCH=true ELASTICSEARCH_PORT=9200 CLUSTERING_TYPE='kubernetesV2' ts-scripts k8s-env",
-        "lint": "eslint",
+        "lint": "DEBUG=typescript-eslint:* eslint --debug",
```
Comment (busma13) [inline package.json:35]: This needs to get changed back

**Review by busma13 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (busma13) [reply package.json:35]:
```suggestion
        "lint": "eslint",
```

**Review by sotojn — APPROVED** [review]
Once Peter's requested changes are in it looks good

**Review by busma13 — APPROVED** [review]

**Pull request merged**